src/test/java/org/apache/drillui/test/framework/testng/unsecure/BaseUnsecureTest.java

```diff
@@ -26,14 +26,14 @@
   @BeforeSuite
   public final void beforeSuite() {
     if (TestProperties.getBool("SECURE_DRILL")) {
-      AuthSteps.login(TestProperties.get("ADMIN_USER1_NAME"), TestProperties.get("ADMIN_USER1_PASSWORD"));
+      new AuthSteps().login(TestProperties.get("ADMIN_1_NAME"), TestProperties.get("ADMIN_1_PASSWORD"));
```

Comment:
Please add fields for steps here and in other occurrences. Even if it occurs once.

Reply:
Done

Reply:
Check other classes as well, where you made changes.

Reply:
Done